nrfu_tests/test_hardware_power_supply_voltage_status.py

```diff
@@ -58,7 +58,11 @@ def test_system_hardware_power_supply_voltage_status(self, dut, tests_definition
             # Skipping test case if the device is vEOS.
             model = version_output.get("modelName")
             if "vEOS" in version_output.get("modelName"):
```

Comment:
Why are we not using model here?
```suggestion
            if "vEOS" in model:
```